src/listeners/scamLinkDetector.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import { Events } from "discord.js";
+import { CustomClient } from "src/client";
+import { env } from "../env";
+
+export default (client: CustomClient) => {
+  console.log("BigBrother listener loaded.");
+
+  client.on(Events.MessageCreate, async (interaction) => {
+    const content = interaction.content;
+
+    const possibleUrls = isUrl(content);
+
+    if (possibleUrls) {
+      for (const url of possibleUrls) {
+        const response = await axios.post(
+          `${env.CHAINPATROL_API_URL}/api/v2/asset/check`,
+          {
+            type: "URL",
+            content: url,
+          }
+        );
+        if (
+          response.data.status === "BLOCKED" ||
+          response.data.status === "UNKNOWN"
+        ) {
+          interaction.react("🚨");
```

Comment:
Does this return a promise? I think we should `await` it

Reply:
https://github.com/chainpatrol/discord-bot/pull/25/commits/97548fe60acf87f96a18e607e726e034a3cf8154